src/server/db/actions/GameAction.ts

```diff
@@ -79,54 +87,118 @@ interface nextEditGame {
 export async function editGame(allData: nextEditGame) {
   await connectMongoDB();
   const data: IEditGame = allData.data;
-  try {
-    if (data && data.themes) {
-      const themeResults = await ThemeModel.find({ id: { $in: data.themes } });
-      if (themeResults.length !== data.themes.length) {
-        throw new InvalidIdGameErrorException(
-          "One of the given themes does not exist.",
-        ); //Using non-null assertion, as if condition should ensure data.tags is non-null
-      }
+  if (data && data.themes) {
+    const themeResults = await ThemeModel.find({ id: { $in: data.themes } });
```

Comment:
This might've been a change I made in main that was merged in but I believe this should be `_id`? I think Lauren already made this change in her branch though